### PR TITLE
GPS, suit storage, and tank dispenser tgui-next

### PR DESCRIPTION
**Pull request by actioninja**
## About The Pull Request

Suit storage looks pretty much the same except I did this because it was easy and looks fun:
image

GPS:
image

Tank Dispenser:
image

## Changelog
:cl:
add: Suit storage, tank dispenser, and GPS have been tgui-nextified
/:cl:

### CODE REVIEW

File: code/datums/components/gps.dm [hunk 1]
```diff
@@ -116,7 +116,6 @@ GLOBAL_LIST_EMPTY(GPS_list)
 			continue
 		var/list/signal = list()
 		signal["entrytag"] = G.gpstag //Name or 'tag' of the GPS
-		signal["area"] = get_area_name(G, TRUE)
```
Comment (nemvar) [inline code/datums/components/gps.dm]: :eyes:

**Comment by Fikou** [comment]
does this fix gps not being able to be resized

**Comment by actioninja** [comment]
Unrelated issue that has already been fixed

File: code/datums/components/gps.dm (hunk 1 above)
Comment (actioninja) [reply code/datums/components/gps.dm]: shh

**Review by stylemistake — APPROVED** [review]

**Comment by GunHog** [comment]
In the GPS interface, is it possible to change the direction text to an arrow? I wanted to do that when I first wrote the TGUI for it, but I did not know how.

**Comment by stylemistake** [comment]
@actioninja actually this is an awesome idea, you can do that with CSS transforms, and `<Icon name="arrow-up" rotation={90} />` would be epic.

**Comment by GunHog** [comment]
Yaaaaaaaaaaay! There are arrows now! They rotate a full 360°, and are brighter if you are close! Thank you for this! 
image

**Comment by PKPenguin321** [comment]
Fancy!

**Comment by stylemistake** [comment]
I decided to clamp max window height, because everyone has different screens, like if there's a window with 1000 height on a 1366x768 screen, you know, it's problematic.

**Comment by stylemistake** [comment]
**Bug:** distance can't go higher than 127.

**Comment by 81Denton** [comment]
conflict

**Comment by actioninja** [comment]
byond limitation, it uses `get_dist` which is capped to 127 for who knows why http://www.byond.com/docs/ref/#/proc/get_dist

**Comment by stylemistake** [comment]
We no longer use that function, then. Begone, 127m limitation!

**Pull request merged**